README.md

```diff
@@ -10,58 +10,35 @@ We developed J2V8 as a high performance engine for our multi-platform mobile too
 
 Building J2V8
 =============
-Building J2V8 requires building both the native parts and the Java library (.jar file). To build the native parts we first build node.js as a library and then statically link J2V8 to that. The Java parts are built with maven.
-
-Building on MacOS
------------------
-```
- sh ./build-node.sh
- sh ./buildJ2V8_macos.sh
- mvn clean verify
-```
-
-Building on Linux
------------------
-```
-export CCFLAGS="${CCFLAGS} -fPIC"
-export CXXFLAGS="${CXXFLAGS} -fPIC"
-export CPPFLAGS="${CPPFLAGS} -fPIC"
-#sh ./build-node.sh
-cp -r /data/jenkins/node .
-cd jni
-g++ -I../node -I../node/deps/v8 -I../node/deps/v8/include \
-    -I../node/src -I /data/jenkins/tools/hudson.model.JDK/jdk-7/include/ \
-    -I /data/jenkins/tools/hudson.model.JDK/jdk-7/include/linux  \
-    com_eclipsesource_v8_V8Impl.cpp -std=c++11 -fPIC -shared -o libj2v8_linux_x86_64.so \
-    -Wl,--whole-archive ../node/out/Release/libnode.a  -Wl,--no-whole-archive \
-    -Wl,--start-group \
-                      ../node/out/Release/libv8_libbase.a \
-                      ../node/out/Release/libv8_libplatform.a \
-                      ../node/out/Release/libv8_base.a \
-                      ../node/out/Release/libv8_nosnapshot.a \
-                      ../node/out/Release/libuv.a \
-                      ../node/out/Release/libopenssl.a \
-                      ../node/out/Release/libhttp_parser.a \
-                      ../node/out/Release/libgtest.a \
-                      ../node/out/Release/libzlib.a \
-                      ../node/out/Release/libcares.a \
-    -Wl,--end-group \
-    -lrt -D NODE_COMPATIBLE=1
-mvn clean verify
-```
-
-This will build J2V8 with node.js support. To disable this support, remove the `-D NODE_COMPATIBLE=1` option.
-
-Building for Android
------------------
-Building J2V8 for Android requires Docker.
-
-```
-./docker/build.sh
-./gradlew assembleRelease
-```
-
-This will build J2V8 as an AAR for API 19 minimum with node.js support.
+Building J2V8 requires building both the native parts and the Java library (.jar/.aar file). To build the native parts we first build node.js as a library and then statically link J2V8 to that. The Java parts are built with maven/gradle.
+
+J2V8 uses a cross-platform, cross-compiling build-system written in Python.
+
+Follow these steps to build J2V8 from source:
+
+1) clone the Node.js source code
+    - `python prepare_build.py`
```

Comment:
It is worth to mention that this works only for python2 (tried with python3 but doesn't work), was there any reason why you choose python over lets say bash scripts? I am just thinking out loud here that not everyone has python2 installed.